war/src/main/webapp/css/style.css

```diff
@@ -650,7 +650,7 @@ div.behavior-loading {
   border-radius: 100%;
   -webkit-animation: sk-scaleout 1.0s infinite ease-in-out;
   animation: sk-scaleout 1.0s infinite ease-in-out;
-  box-shaodw:#fff 0 0 0 10px
+  box-shadow:#fff 0 0 0 10px
```

Comment:
> (I obviously don't know -- but the current line is just bloat)

It's up to you as submitter to show what impact this has, which is what I guess Daniel was getting at.

Anyway, I tested this change, and it just adds a fairly subtle white halo around the circle animation:
![image](https://cloud.githubusercontent.com/assets/138615/21999224/49fd8634-dc39-11e6-847b-d712dd8c7cba.png)